### PR TITLE
Qt4 fsarchiver

### PR DESCRIPTION
**Pull request by necrose99**
updated to the 3 currently supported versions (SF.net) 

ARCH TESTING for keywords ie ARM/MIPS/PPC/ Gentoo BSD etc would be a plus. 
keywords not currently known to run.

### CODE REVIEW

**Comment by NP-Hardass** [comment]
Please use "repoman commit" instead of "git commit".   Generally speaking, we do not support commits made without using "repoman commit."  There are a number of issues stated https://qa-reports.gentoo.org/output/gentoo-ci/f2b0b9a/15.html#l1720, but you may find running "repoman full" in the package directory more useful in generating QA warnings to help you clean this PR up.

**Comment by necrose99** [comment]
nuts i tried to isolate my pull on a branch  to just these.... I'll just flipping bug request this is getting tedious else web ui wont let me just add 1-3 files at a go.

**Comment by gktrk** [comment]
The commit messages do not follow the proper documented format [1]. Using repoman commit helps in avoiding things like this.

Is there a particular reason why a6ab8a9 and c61e59a are separate commits? Looks like they should be made into a single commit.

A small nitpick: users are encouraged to sign their commits, though this is not a hard requirement AFAIK.

[1] https://wiki.gentoo.org/wiki/Gentoo_git_workflow#Commit_message_format

File: app-backup/qt4-fsarchiver/qt4-fsarchiver-0.6.19.15.ebuild [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+#epatch "${DISTDIR}"starter/Gentoo-qt4-fsarchiver-pro.diff
+# fix .desktop file ~I dont sed well yet for all 3 desktop files,
+sed -i -e '/Encoding/d' starter/*.desktop || die "sed on qt4-fsarchiver.desktop failed"
+	# fix .desktop file
+	sed -e '/Encoding/d' -i starter/*.desktop || die
+	# fix icon installation location
+	sed -i \
+		-e "/icon.path/s:app-install/icons:${PN}:" "${PN}.pro" \
+	|| die "sed on ${PN}.pro failed"
+}
```
Comment (gktrk) [inline app-backup/qt4-fsarchiver/qt4-fsarchiver-0.6.19.15.ebuild]: Trailing whitespace after '}'

File: app-backup/qt4-fsarchiver/qt4-fsarchiver-0.6.19.14.ebuild [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+#epatch "${DISTDIR}"starter/Gentoo-qt4-fsarchiver-pro.diff
+# fix .desktop file ~I dont sed well yet for all 3 desktop files,
+sed -i -e '/Encoding/d' starter/*.desktop || die "sed on qt4-fsarchiver.desktop failed"
+	# fix .desktop file
+	sed -e '/Encoding/d' -i starter/*.desktop || die
+	# fix icon installation location
+	sed -i \
+		-e "/icon.path/s:app-install/icons:${PN}:" "${PN}.pro" \
+	|| die "sed on ${PN}.pro failed"
+}
```
Comment (gktrk) [inline app-backup/qt4-fsarchiver/qt4-fsarchiver-0.6.19.14.ebuild]: Trailing whitespace after '}'

File: app-backup/qt4-fsarchiver/qt4-fsarchiver-0.6.19.13.ebuild [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+#epatch "${DISTDIR}"starter/Gentoo-qt4-fsarchiver-pro.diff
+# fix .desktop file ~I dont sed well yet for all 3 desktop files,
+sed -i -e '/Encoding/d' starter/*.desktop || die "sed on qt4-fsarchiver.desktop failed"
+	# fix .desktop file
+	sed -e '/Encoding/d' -i starter/*.desktop || die
+	# fix icon installation location
+	sed -i \
+		-e "/icon.path/s:app-install/icons:${PN}:" "${PN}.pro" \
+	|| die "sed on ${PN}.pro failed"
+}
```
Comment (gktrk) [inline app-backup/qt4-fsarchiver/qt4-fsarchiver-0.6.19.13.ebuild]: Trailing whitespace after '}'

File: app-backup/qt4-fsarchiver/qt4-fsarchiver-0.6.19.13.ebuild [hunk 4]
```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit qmake-utils eutils versionator  #qt5-build unknow test case,
```
Comment (gktrk) [inline app-backup/qt4-fsarchiver/qt4-fsarchiver-0.6.19.13.ebuild]: looks like a minor typo: s/unknow/unknown/

File: app-backup/qt4-fsarchiver/qt4-fsarchiver-0.6.19.14.ebuild [hunk 5]
```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit qmake-utils eutils versionator  #qt5-build unknow test case,
```
Comment (gktrk) [inline app-backup/qt4-fsarchiver/qt4-fsarchiver-0.6.19.14.ebuild]: looks like a minor typo: s/unknow/unknown/

File: app-backup/qt4-fsarchiver/qt4-fsarchiver-0.6.19.15.ebuild [hunk 6]
```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit qmake-utils eutils versionator  #qt5-build unknow test case,
```
Comment (gktrk) [inline app-backup/qt4-fsarchiver/qt4-fsarchiver-0.6.19.15.ebuild]: looks like a minor typo: s/unknow/unknown/

**Pull request closed without merging**